build.sbt

```diff
@@ -23,7 +23,7 @@ ThisBuild / githubWorkflowBuildPreamble +=
     List("decryptSecret gcs/src/test/resources/bad-auth-file.json.enc"),
     name = Some("Decrypt bad gcp service account json key"))
 
-val AwsSdkVersion = "2.15.34"
+val AwsSdkVersion = "2.16.21"
```

Comment:
This needs to be bumped because of netty version change, right? Needs to be done in other libs using the aws sdk as well.

And I think something similar needs to happen for azure as well because they also use netty.

It can be quite a pain to get all of the netty versions line up across all libs. Depending on how far we are with this, it may be easier to roll back https://github.com/precog/quasar-datasource-url/pull/763 et al and publish our own fix directly on top of http4s `0.21.25` so that nothing related to netty changes.

Reply:
Yeah, we came to the same conclusion and are giving that a go.